process_testsuite.go

```diff
@@ -117,13 +117,17 @@ func (v *Venom) runTestCases(ctx context.Context, ts *TestSuite) {
 			tc.Duration = tc.End.Sub(tc.Start).Seconds()
 		}
 
+		skippedSteps := 0
 		for _, testStepResult := range tc.TestStepResults {
 			if testStepResult.RangedEnable {
 				hasRanged = true
 			}
 			if testStepResult.Status == StatusFail {
 				hasFailure = true
 			}
+			if testStepResult.Status == StatusSkip {
+				skippedSteps++
+			}
 		}
 
 		if verboseReport || hasRanged {
```

Comment:
Not sure if it's because of my environment, but the newline seems to be printed too late when I run venom in verbose mode:
![image](https://user-images.githubusercontent.com/22963968/196247387-5dcedf17-29ec-41d5-974a-dbc3cb70636f.png)

In my tests I moved this block to line 114 (just before `v.runTestCase(ctx, ts, tc)`) and it seems to yield the correct output.

Since the output from https://github.com/ovh/venom/issues/583#issuecomment-1243679828 looked correct, I didn't touched it, but just wanted to make a note about it

Reply:
you're right, fixed.